docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,169 @@
+
+[source,bash]
+----
+kubectl create namespace coherence-operator
+----
+
+Istio Sidecar AutoInjection is done automatically if you label coherence-operator namespace with istio-injection. If you want to implement manual injection with the deployments then you need to use istioctl kube-inject.
+
+[source,bash]
+----
+kubectl label namespace coherence-operator istio-injection=enabled
+----
+
+After the namespace is labeled, you can install the operator using your preferred method in the Operator https://oracle.github.io/coherence-operator/docs/latest/#/installation/01_installation[Installation Guide].
+
+After installed operator, use the following command to confirm the operator is running:
+
+[source,bash]
+----
+kubectl get pods -n coherence-operator
+----
+
+== Creating a Coherence cluster with Istio support
+
+You can configure your cluster to run with Istio automatic sidecar injection enabled. Before creating your cluster, create the namespace in which you want to run the cluster and label it for automatic injection.
+
+[source,bash]
+----
+kubectl create namespace coherence-cluster
+kubectl label namespace coherence-cluster istio-injection=enabled
+----
+
+There is no other requirements to run Coherence in Istio environment.
+
+The following is an example that creates a cluster named example-cluster:
+
+example.yaml
+[source,bash]
+----
+# Example
+apiVersion: coherence.oracle.com/v1
+kind: Coherence
+metadata:
+  name: example-cluster
+----
+
+[source,bash]
+----
+$ kubectl -n coherence-cluster apply -f example.yaml
+----
+
+After you installed the Coherence cluster, run the following command to view the pods:
+
+[source,bash]
+----
+$ kubectl -n coherence-cluster get pods
+
+NAME                                             READY   STATUS    RESTARTS   AGE
+example-cluster-0                                2/2     Running   0          45m
+example-cluster-1                                2/2     Running   0          45m
+example-cluster-2                                2/2     Running   0          45m
+----
+
+You can see that 3 members in the cluster are running with 3 pods. 2/2 in READY column means that there are 2 containers running in each Pod. One is Coherence member and the other is Envoy Proxy.
+
+== Prometheus
+
+You can use Prometheus with Istio to record metrics that track the health of Istio and of Coherence cluster within the service mesh and view them through Granfana. 
+
+== Traffic Management
+
+Istio provides traffic management capabilities, including the ability to visualize traffic in Kiali. You do not need to change your applications to use this feature. The Istio proxy (envoy) sidecar that is injected into your pods provides it. The image below shows an example with traffic flowing: In from the Istio gateway on the left, to a cluster called example-cluster in the coherence-example namespace.
```

Comment:
You are conflating two things, traffic management and visualisation and then you only really talk about visualisation. Traffic management is a much broader subject about how you Configure Istio to route traffic to different services, enable TLS etc. You can still mention traffic management and how it applies to Coherence to manage Extend connections, REST, gRPC clients etc, and whether there are any specific things to bear in mind with Coherence when using Istio traffic management.

Reply:
Changed to "Traffic Visualization" and stated Coherence cluster traffic is not visible to Istio.